src/Button/Button/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const Button = ({ className, color, size, disabled, children }) => {
```

Comment:
Consider having an extra boolean prop for `block`. Right now, from looking at the stories, you're including it with the `size` prop but whether a button is inline or block is different from how large it is.